### DISCUSSION TITLE
How can I close all active ssh & mosh sessions?

### DISCUSSION THREAD

**Discussion by crustulumtheoneandonly** [blinksh/blink#1542] [Q&A] [answered]
First of all: Great App!!

I was wondering if there already is some way to quickly close all open sessions?

currently the number of my sessions increases fast and after a few days/weeks I just want to close everything but need to do it on every session individual (mosh and ssh)

is there already a shortcut or could you add a button in the config?

**Comment by crustulumtheoneandonly** [comment]
Maybe an extended Option of the „exit“ command like:

`exit —all-sessions -y`

(`-y` For dialog of approval)

any thoughts on this?

**Comment by carloscabanero** [comment, marked as answer]
I guess that is a great sign! We will consider this as we redesign things in the future, but here are a few tips to make things easier:
- If you have multiple windows, it should be easy to just close the "window" instead of the individual sessions. On iPhone this may not work, but on the iPad it should be simple. In order for Blink not to "restore" everything, just make sure there is always a blank window where you can start over.
- On the iPhone, with a three finger tap you will display the control menu. From there it should be "quick" to just press the "close" button recursively for all tabs you wanna close, as it is quite consistent in position, etc...

**Reply by crustulumtheoneandonly** [reply]
Great! Thanks didn’t know that feature!

works for me 👍

thanks for the quick reply
